.circleci/config.yml

```
@@ -174,6 +188,23 @@ jobs:
           paths:
             - ~/.m2
           key: *mvn_cache_key
+  generate-dags:
+    docker: *docker
+    steps:
+      - checkout
+      - *restore_venv_cache
+      - *build
+      - *restore_mvn_cache
+      - *java_deps
+      - *attach_generated_sql
+      - *copy_generated_sql
+      - run:
+          name: Generate DAGs
+          command: PATH="venv/bin:$PATH" script/generate_airflow_dags
+      - persist_to_workspace:
+          root: /tmp/workspace
+          paths:
+            - generated-sql
```

```suggestion
      - persist_to_workspace:
          root: /tmp/workspace
          paths:
            - generated-sql
```

We already have the `generate-sql` task persisting to this exact same location. Do we know how that behaves? I suppose if one of the tasks that depends on `generate-sql` were to pick up the content produced by `generate-dags`, it would still be fine because it's just additive.

It would be worth adding a comment here that this is overwriting the content produced by `generated-sql`, though, since that behavior might be surprising.

---

I added a comment for clarification.